### PR TITLE
Update DG - Bulk Increment Salary

### LINKED ISSUE #126: Update DG - Bulk Increment Salary

### PR DESCRIPTION
**Pull request by Goh-Li-Ting**
Add implementation details, design considerations, sequence diagram and use case for bulk increment salary
closes #126

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -154,6 +154,44 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Increment Feature
+
+#### Implementation
+
+The increment feature is facilitated by the following operations:
+* `Model#getFilteredPersonList()` — Returns the list of employees in the filtered list.
```
Comment (Weiennn) [inline docs/DeveloperGuide.md:162]: small update to the method; 
Model#getFilteredPersonList() has been changed to Model#getSortedFilteredPersonList()

Comment (Weiennn) [reply docs/DeveloperGuide.md:162]: Do remember to change the method name downstream also

**Review by Weiennn — APPROVED** [review]
LGTM

**Comment by Goh-Li-Ting** [comment]
Modified the sequence diagram to omit details on parsing command

**Review by aexolate — APPROVED** [review]
LGTM

**Pull request merged**